extensions/kind/src/create-cluster.spec.ts

```diff
@@ -117,15 +117,26 @@ test('expect error if Kubernetes reports error', async () => {
 });
 
 test('check cluster configuration generation', async () => {
-  const conf = getKindClusterConfig('k1', 80, 443);
+  const conf = getKindClusterConfig('k1', 80, 443, 'image:tag');
   expect(conf).to.contains('name: k1');
   expect(conf).to.contains('hostPort: 80');
   expect(conf).to.contains('hostPort: 443');
+  expect(conf).to.contains('image: image:tag');
+});
+
+test('check cluster configuration empty string image', async () => {
+  const conf = getKindClusterConfig(null, null, null, '');
```

Comment:
you need to use `undefined` instead of `null` we're blacklisting the usage of `null` in the codebase